docs/content/mobile/android/local-deploy.md

```diff
@@ -9,15 +9,21 @@ Before reading this document, make sure you are familiar with:
 
 ## Prepare your project to connect to a local deployment {#prepare}
 
-To connect your Android device to a local deployment, you need to prepare its configuration. Follow the steps below to ensure any local deployment you launch via the Unity Editor is correctly set up.
+To connect your Android device to a local deployment, you need to prepare its configuration. Follow the steps below to ensure any local deployment you launch via your Unity Editor is correctly set up.
 
-1. Open your project in the Unity Editor.
+1. Open your project in your Unity Editor.
 1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
 1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
 1. Select **Save** and close the window.
 
+Create and add a MonoBehaviour script to your Android client-worker GameObject. To do this:
```

Comment:
this is still the old version, let's try to make them the same for local and cloud as far as possible

Reply:
The format of adding the MonoBehaviour to the GameObject has been updated on cloud-deploy.md to match the format of local.
The first three steps are similar but cloud then provides more information on adding the authentication token you created to the `DevelopmentAuthToken` field.